gateway/public/index.html

```diff
@@ -0,0 +1,318 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="A unified gateway solution designed to seamlessly connect and manage multiple networks through a single, streamlined interface."
```

Comment:
I think the different description should be a bit more specific to Autonomys, `manage multiple networks` is a bit vague. We don't really use this endpoint to `manage networks` but rather to access file regardless if they are stored on mainnet or testnet.
```suggestion
      content="A unified gateway solution designed to seamlessly connect and access files permanently stored on Autonomys Mainnet&Tesnet through a single, streamlined interface."
```
Please also change it in other location